dotnetUno/__change__.Skia.Gtk/__change__.Skia.Gtk.csproj

```diff
@@ -19,11 +19,11 @@
     <Compile Include="..\Utils\*.cs" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="5.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="5.0.0" />
-    <PackageReference Include="Uno.UI.Skia.Gtk" Version="4.8.33" />
-    <PackageReference Include="Uno.UI.RemoteControl" Version="4.8.33" Condition="'$(Configuration)'=='Debug'" />
-    <PackageReference Include="Uno.UI.Adapter.Microsoft.Extensions.Logging" Version="4.8.33" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="7.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="7.0.0" />
+    <PackageReference Include="Uno.UI.Skia.Gtk" Version="4.10.13" />

```

Comment:
this should be `4.10.23` not?
https://www.nuget.org/packages/Uno.UI.Skia.Gtk/

Reply:
Interesting, I got the version from here: [https://github.com/unoplatform/uno/releases](https://github.com/unoplatform/uno/releases). Do you know if there is any specific reason for them not to publish every version on Gihub?